resources/views/banking/accounts/mail.blade.php

```diff
@@ -1,9 +1,20 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <title>Bank Account</title>
-</head>
-<body>
-    Bank Account
-</body>
-</html> 
+<style>
+    table{
+        width: 100% !important;
+    }
+</style>
+
+@component('mail::message')
+Good day,
+
+This is your bank account details.
+
+@component('mail::table')
+Account Name | Account Number  | Bank Branch | Account Type | Account Status | Account Balance |
+|:--------|:-------------|:-------|:---------|:-------| :-------|:-------|:-------|
+| {{$bank_account['account_name']}} | {{$bank_account['bank_account_number']}}  | {{$bank_account['bank_branch']}} | {{$bank_account['bank_account_type']}} | {{$bank_account['status']}} | {{$bank_account['current_balance']}} |
```

Comment:
Make it something like this:
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |

```
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |
```